### DISCUSSION TITLE
Using Links - Prefill Data -  Documentation and Possible Bug?

### DISCUSSION THREAD

**Discussion by Lu-H** [kimai/kimai#2903] [Questions & Answers] [answered]
We are a small engineering office that wants to switch from excel to kimai to keep track of the time spent on different projects. We managed to install version 1.15.5 on our ionos website. 
Right now it seems to be exactly what we are looking for. So I want to use this post also to thank you very much for the hard work you put into the software!
I'm new to github - Please excuse if I asked the question in the wrong place.

**Is your feature request related to a problem? Please describe.**
We are using Microsoft Teams for project work and want to integrate kimai as a website in the corresponding project channel. I saw in one issue the possibility to prefill data, e.g. https://kimai.website.de/de/timesheet/create?project=1&activity=2
Is there a place were the keywords  like "project" and "activity" or the syntax of the links are documented? Are there other keywords?

Now the bug (or feature request):
The syntax does not work for multiple users within the link - https://kimai.website.de/de/team/timesheet/create_mu?project=1&activity=2
We would love this feature, because we often have internal meetings and usually just one person afterwards creates the record for all participants. 

**Describe the solution you'd like**
Posibility to prefill data for create-form "multiple users"
https://kimai.website.de/de/team/timesheet/create_mu?project=1&activity=2

**Describe alternatives you've considered**
none yet

**Comment by kevinpapst** [comment, marked as answer]
`project` , `activity`, `description`, `tags`, `begin` and `end` are the GET parameter that are supported in the normal create form.

If you use Kimai in a company, you could sponsor that feature to support these/more fields in the create-for-multiple-users form.

**Reply by Lu-H** [reply]
Thank you very much for your very quick support! We will get back to this offer, when we decided to leave our testing-status. 
Can you please point us in the right direction which permission is needed to be set to see all time sheets in one project/ team (or to create a time sheet for multiple users)? Right now I only see the (view_all_data) permission, which allows to see and fill in multiple users in https://kimai.website.de/de/team/timesheet/create_mu
Or is this permission not yet developed and needs to be sponsored?

**Reply by kevinpapst** [reply]
That should be the standard setting for teamleads. Be aware: the user role "teamlead" only gives the permission to use the functionality. The user needs to be the "Teamlead" of the team as well. Only if he is the Teamlead of the team, he will be able to see the data of team members (or select them in dropdown boxes). The permissions that end with `_other_timesheet` are important especially create_other_timesheet, view_other_timesheet and edit_other_timesheet).

See https://www.kimai.org/documentation/permissions.html

**Reply by Lu-H** [reply]
Thank you very much! Now I begin to understand the settings. Originally, I was thinking about giving all regular users the permissions with `_other_timesheet` but didn't think of the assigned team-functionality.
